docs/probes.md

```diff
@@ -26,105 +26,114 @@ Make sure to provide enough delay and failureThreshold for the VM and the agent
 
 ### Example
 
-**Note**: The Fedora image used in this example does not have qemu-guest-agent available by default.
-We need to install and enable it via cloud-init as shown in the example below.
+The Fedora image used in this example does have qemu-guest-agent available by default. Nevertheless, in
+case qemu-guest-agent is not installed, it will be installed and enabled via cloud-init as shown in the example below. 
+Also, cloud-init assigns the proper SELinux context, i.e. virt_qemu_ga_exec_t, to the `/tmp/healthy.txt` file. 
+Otherwise, SELinux will deny the attempts to open the `/tmp/healthy.txt` file causing the probe to fail.
+
+> Note:  If SELinux is not installed in your container disk image, the command `chcon` should be removed from the VM
+> manifest shown below. Otherwise, the `chcon`  command will fail.
 
 1. Create VM manifest
 
 ```yaml
-# /tmp/probe-test.vm.yaml
-apiVersion: kubevirt.io/v1alpha3
+# /tmp/probe-test.yaml
+apiVersion: kubevirt.io/v1
 kind: VirtualMachine
 metadata:
-  name: probe-test
-  namespace: default
   labels:
-    app: probe-test
-    vm.kubevirt.io/name: probe-test
-    kubevirt.io/domain: probe-test
+    kubevirt.io/vm: readiness-probe-vm
+  name: readiness-probe
 spec:
   running: false
   template:
     metadata:
       labels:
-        vm.kubevirt.io/name: probe-test
-        kubevirt.io/domain: probe-test
+        kubevirt.io/domain: readiness-probe
+        kubevirt.io/vm: readiness-probe
     spec:
-      readinessProbe:
-        exec:
-          command:
-          - cat
-          - /tmp/ready.txt
-        failureThreshold: 10
-        initialDelaySeconds: 120
-        periodSeconds: 10
-        # Note that timeoutSeconds value does not have any impact before K8s v1.20.
-        timeoutSeconds: 5
       domain:
         cpu:
           cores: 1
           sockets: 1
           threads: 1
         devices:
           disks:
-            - disk:
+            - name: containerdisk
+              disk:
                 bus: virtio
-              name: cloudinitdisk
-            - bootOrder: 1
+            - name: cloudinitdisk
               disk:
                 bus: virtio
-              name: rootdisk
           interfaces:
```

Comment:
You can drop `interfaces` and `networks` if all you want is an interface on the default network. One will be added automatically.